### PR TITLE
Fix typo in plugin description

### PR DESCRIPTION
**Pull request by foobar13372**

### CODE REVIEW

**Comment by foobar13372** [comment]
I do not know what this CLA signature thing is that makes this fail. Consider my commit public domain. I'll close it if bothered with bureaucratism.

File: plugins/analysis-icu/build.gradle [hunk 1]
```diff
@@ -20,7 +20,7 @@ import de.thetaphi.forbiddenapis.gradle.CheckForbiddenApis
  */
 
 esplugin {
-  description 'The ICU Analysis plugin integrates Lucene ICU module into elasticsearch, adding ICU relates analysis components.'
+  description 'The ICU Analysis plugin integrates Lucene ICU module into elasticsearch, adding ICU related analysis components.'
```
Comment (jrodewig) [inline plugins/analysis-icu/build.gradle]:
```suggestion
  description 'The ICU Analysis plugin integrates Lucene ICU module into elasticsearch, adding ICU-related analysis components.'
```

**Comment by jrodewig** [comment]
hi @foobar13372 

Unfortunately, we can't merge the pull request until you sign the CLA here:
https://www.elastic.co/contributor-agreement

Thanks for your contribution!

**Comment by foobar13372** [comment]
As announced: bothered with bureaucratism -> closed.

**Pull request closed without merging**